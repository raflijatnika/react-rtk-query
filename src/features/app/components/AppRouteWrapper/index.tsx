// React
import { memo } from 'react'

// React Router DOM
import { Outlet } from 'react-router-dom'

const AppRouteWrapper = memo(() => {
  return <Outlet />
})

AppRouteWrapper.displayName = 'AppRouteWrapper'

export { AppRouteWrapper }
