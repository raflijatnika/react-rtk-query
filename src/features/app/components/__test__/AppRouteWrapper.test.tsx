// Test
import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'

// Components
import { AppRouteWrapper } from '../AppRouteWrapper'

it('AppRouteWrapper should render', () => {
  render(<AppRouteWrapper />)
})

it('AppRouteWrapper should have snapshot', () => {
  const tree = renderer.create(<AppRouteWrapper />).toJSON()
  expect(tree).toMatchSnapshot()
})
