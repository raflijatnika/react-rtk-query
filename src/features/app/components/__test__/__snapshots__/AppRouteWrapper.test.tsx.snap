// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRouteWrapper should have snapshot 1`] = `null`;
